backend/pkg/config/config.go

```diff
@@ -141,6 +144,22 @@ func Parse(args []string) (*Config, error) {
 		}
 	}
 
+	if config.OidcSkipTLSVerify {
+		logger.Log(logger.LevelWarn, nil, nil, "oidc-skip-tls-verify is set, this is not safe for production")
+	}
+
+	if config.OidcCAFile != "" {
+		caFile, err := os.ReadFile(config.OidcCAFile)
+		if err != nil {
+			return nil, fmt.Errorf("error reading oidc-ca-file: %w", err)
+		}
+		// check if the file is a valid PEM file
+		caCertPool := x509.NewCertPool()
+		if !caCertPool.AppendCertsFromPEM(caFile) {
```

Comment:
Can you please rename the caFile to caFileContents? Currently it shares the same convention as the config.OidcCAFile... which lead me to believe from just reading this part that it was a filename.

It might also be worth renaming config.OidcCAFile to config.OidcCAFilePath to be more explicit.